baksnapper.sh

```diff
@@ -489,6 +505,10 @@ function incremental-backup {
         $receiver remove-broken-snapshot "$dest_root" "$1"
         error "Failed to send snapshot!"
     fi
+    if [ $p_link -eq 1 ]
```

Comment:
Change this to `[[ $p_link -eq 1 ]]`
Getting `[: -eq: unary operator expected` when running without `--link` option